src/stories/Quiz/Component/QuizResults/QuizResults.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { expect } from '@storybook/jest';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { within } from '@storybook/testing-library';
+import ResultContainer from '../../../../components/ResultContainer/ResultContainer';
+import QuizResultsDecorator from './QuizResultsDecorator';
+
+const meta: Meta<typeof ResultContainer> = {
+  title: 'Quiz/Results/QuizResultsPage',
+  component: ResultContainer,
+  argTypes: {},
+};
+
+export default meta;
+type Story = StoryObj<typeof ResultContainer>;
+
+const resultsPageOptions = {
+  onAddToCartClick: () => {},
+  resultCardRegularPriceKey: 'price',
+  resultCardSalePriceKey: 'salePrice',
+};
+
+export const QuizResultsPage: Story = {
```

Comment:
I can't really tell what but something looks off here 🤔 Maybe the images and titles
![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/f7d5fcf8-0a56-41cb-8247-0bfca70fd7bf)